vault/tests/test_vault.py

```diff
@@ -637,6 +651,7 @@ def test_token_renewal(self, caplog, aggregator, dd_run_check, instance, global_
     @auth_required
     def test_auth_needed_but_no_token(self, aggregator, dd_run_check, instance, global_tags):
         instance = instance()
```

Comment:
Why is this test not using the openmetrics V2 implementation?

Reply:
I think I encountered some problems with some of the tests when trying to run them with the V2 implementation so I simply skipped those to move forward, and forgot to get back to them. I'll have another look at all the tests that are currently only exercising v1.